mozilla-plugin stuff is actually in /unstable not in /stable

Causes bugs like http://bugs.gentoo.org/257803

---
--- xulrunner/installer/mozilla-plugin.pc.in~	2009-06-26 08:37:34.458635913 -0500
+++ xulrunner/installer/mozilla-plugin.pc.in	2009-06-26 08:38:13.623656097 -0500
@@ -1,9 +1,10 @@
 prefix=%prefix%
 sdkdir=%sdkdir%
 includedir=%includedir%
+includetypes=unstable
 
 Name: Mozilla Plug-In API
 Description: Mozilla Plug-In API
 Version: %MOZILLA_VERSION%
 Requires: %NSPR_NAME% >= %NSPR_VERSION%
-Cflags: -I${includedir}/stable -DXP_UNIX %ojidef%
+Cflags: -I${includedir}/${includetypes} -DXP_UNIX %ojidef%
